build/build-bundle.js

```diff
@@ -124,35 +123,48 @@ async function browserifyFile(entryPath, distPath) {
     writeStream.on('finish', resolve);
     writeStream.on('error', reject);
 
-    bundleStream.pipe(writeStream);
+    bundleStream
+      // Extract the inline source map to an external file.
+      .pipe(exorcist(`${distPath}.map`))
+      .pipe(writeStream);
   });
 }
 
 /**
- * Minimally minify a javascript file, in place.
+ * Minify a javascript file, in place.
  * @param {string} filePath
  */
 function minifyScript(filePath) {
-  const opts = {
-    compact: true, // Do not include superfluous whitespace characters and line terminators.
-    retainLines: true, // Keep things on the same line (looks wonky but helps with stacktraces)
-    comments: false, // Don't output comments
-    shouldPrintComment: () => false, // Don't include @license or @preserve comments either
-    plugins: [
-      'syntax-object-rest-spread',
-      'syntax-async-generators',
-    ],
-    // sourceMaps: 'both'
-  };
+  const result = terser.minify(fs.readFileSync(filePath, 'utf-8'), {
+    output: {
+      comments: /^!/,
+      // @ts-ignore - terser types are whack-a-doodle wrong.
+      max_line_len: /** @type {boolean} */ (1000),
+    },
+    // The config relies on class names for gatherers.
+    keep_classnames: true,
+    // Runtime.evaluate errors if function names are elided.
+    keep_fnames: true,
+    sourceMap: {
+      content: JSON.parse(fs.readFileSync(`${filePath}.map`, 'utf-8')),
+      url: path.basename(`${filePath}.map`),
+    },
+  });
+  if (result.error) {
+    throw result.error;
+  }
 
-  // Add the banner and modify globals for DevTools if necessary
-  let minified = BANNER + babel.transformFileSync(filePath, opts).code;
-  if (isDevtools(filePath)) {
-    assert.ok(minified.includes('\nrequire='), 'missing browserify require stub');
-    minified = minified.replace('\nrequire=', '\nglobalThis.require=');
-    assert.ok(!minified.includes('\nrequire='), 'contained unexpected browserify require stub');
+  // Add the banner and modify globals for DevTools if necessary.
+  // This will mess up the source map for the first line, but we don't ship
+  // the map to DevTools, so that's fine.
```

Comment:
what does that leave that gets a source map?

Reply:
I just removed this comment since we aren't shipping any maps in CDT or LR. I put the default map generation back to false. can set to `true` whenever we want to get a bundle viz.